<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterNodeInfoWidget</class>
 <widget class="QFrame" name="CharacterNodeInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>799</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Character Node Information</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>647</width>
        <height>793</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="3" column="0">
           <widget class="QComboBox" name="maskBox">
            <property name="toolTip">
             <string>Select a mask to edit or delete.</string>
            </property>
            <property name="styleSheet">
             <string notr="true">margin-left: 5px</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLineEdit" name="maskName">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Name of the mask. This will be available to apply when playing clips later.</string>
            </property>
            <property name="styleSheet">
             <string notr="true">margin-left: 5px</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="4">
           <widget class="QTreeWidget" name="jointTree">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border-top: 1px solid palette(midlight);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::EditKeyPressed</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="indentation">
             <number>20</number>
            </property>
            <property name="rootIsDecorated">
             <bool>true</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>false</bool>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>false</bool>
            </property>
            <attribute name="headerCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Joint</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Weight</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="4" column="2" colspan="2">
           <widget class="QCheckBox" name="renderSkeleton">
            <property name="toolTip">
             <string>Check to debug render the skeleton.</string>
            </property>
            <property name="styleSheet">
             <string notr="true">margin-right: 2px;</string>
            </property>
            <property name="text">
             <string>Render Skeleton</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="2">
           <widget class="QPushButton" name="deleteMaskButton">
            <property name="toolTip">
             <string>Delete selected mask</string>
            </property>
            <property name="styleSheet">
             <string notr="true">margin-right: 2px;</string>
            </property>
            <property name="text">
             <string>Delete mask</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="displayJoints">
            <property name="text">
             <string>Display</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="newMaskButton">
            <property name="toolTip">
             <string>Create new mask with a placeholder name.</string>
            </property>
            <property name="styleSheet">
             <string notr="true">margin-right: 2px;</string>
            </property>
            <property name="text">
             <string>New mask</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
